include/zephyr/net/dns_resolve.h

```diff
@@ -89,49 +89,49 @@ enum dns_query_type {
  */
 struct dns_addrinfo {
 	struct sockaddr ai_addr;
-	socklen_t       ai_addrlen;
-	uint8_t            ai_family;
-	char            ai_canonname[DNS_MAX_NAME_SIZE + 1];
+	socklen_t ai_addrlen;
+	uint8_t ai_family;
+	char ai_canonname[DNS_MAX_NAME_SIZE + 1];
```

Comment:
This seems to be unrelated change, please remove this, or add this to new commit if you think this should be there.